janusgraph-core/src/main/java/org/janusgraph/graphdb/tinkerpop/optimize/AdjacentVertexFilterOptimizerStrategy.java

```diff
@@ -14,6 +14,7 @@
 
 package org.janusgraph.graphdb.tinkerpop.optimize;
 
+import org.apache.tinkerpop.gremlin.process.traversal.step.util.EmptyStep;
```

Comment:
Move this import on the line 22. I.e. after:
`import org.apache.tinkerpop.gremlin.process.traversal.Step;`

I know that it look strange but current style is to use tinkerpop imports below janusgraph imports. I don't know that is the purpose of this style, I don't get it, but it is used right now.

Reply:
Ok, I resort imports